src-backend/api/views.py

```diff
@@ -37,6 +39,33 @@ def get_queryset(self):
         user = self.request.user
         return models.Page.objects.filter(procedure__owner_id__exact=user.id)
 
+    def get_serializer(self, *args, **kwargs):
+        if "data" in kwargs:
+            data = kwargs["data"]
+
+            if isinstance(data, list):
+                kwargs["many"] = True
+
+        return super(PageViewSet, self).get_serializer(*args, **kwargs)
+
+    @list_route(methods=["PATCH"])
+    def partial_bulk_update(self, request, *args, **kwargs):
+        partial = kwargs.pop('partial', False)
+
+        queryset = self.filter_queryset(self.get_queryset)
+
+        serializer = self.get_serializer(
+            instance = queryset,
+            data = json.loads(request.body),
```

Comment:
Would returning a `HTTP_406_NOT_ACCEPTABLE` be what we want if the body is empty? The only time this route gets called is when we re-order, so there *should* be a body, but I can still check.